src/main/java/Duke.java

```diff
@@ -1,10 +1,134 @@
+import java.util.Scanner;
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
+
+    private static final Scanner SCANNER = new Scanner(System.in);
+    private static final char INPUT_COMMENT_MARKER = '#';
+    private static final int CAPACITY = 99;
+    private static TaskManager tasks;
```

Comment:
Cool usage of TaskManager class to handle Tasks. However, the naming of the variable might be too similar with the  array of tasks within the class.